flake.nix

```diff
@@ -26,6 +26,7 @@
         {
           devShells.default = pkgs.mkShell {
             nativeBuildInputs = with pkgs; [
+              http-server
```

Comment:
use python3 -m http.server

Reply:
That sets caching headers on 1970 which requires force reload every time and I didn't find a way to configure that.